src/zenml/integrations/seldon/services/seldon_deployment.py

```diff
@@ -183,12 +239,35 @@ def _get_seldon_deployment_labels(self) -> Dict[str, str]:
         Returns:
             The labels for the Seldon Core deployment.
         """
-        return dict(
-            zenml_pipeline_name=self.config.pipeline_name or "",
-            zenml_pipeline_run_id=self.config.pipeline_run_id or "",
-            zenml_pipeline_step_name=self.config.pipeline_step_name or "",
-            zenml_service_uuid=str(self.uuid),
-        )
+        labels = self.config.get_seldon_deployment_labels()
+        labels["zenml.service_uuid"] = str(self.uuid)
+        SeldonClient.sanitize_labels(labels)
+        return labels
+
+    @classmethod
+    def create_from_deployment(
+        cls, deployment: SeldonDeployment
+    ) -> "SeldonDeploymentService":
+        """Recreate a Seldon Core service from a Seldon Core
+        deployment resource and update their operational status.
```

Comment:
Wait isn't this the same class as in line 113?

Reply:
Similar, yes, but not the same. This one creates an entire service from the information stored in the SeldonDeployment resource. The other one (line 113) creates just the configuration part.

In fact, the function at line 113 is called in this one, further down.

Reply:
my bad